doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2139,6 +2139,19 @@ NCSDK-19942: HID samples do not work with Android 13
   * :ref:`fast_pair_input_device`
   * Fast Pair configurations of the :ref:`nrf_desktop` application
 
+.. rst-class:: wontfix v2-7-0 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+
+NCSDK-26669: Reconnection with HID devices that have privacy enabled may not work on Android 14
+  This issue affects samples and applications that are set up for the HIDS use case and have the Bluetooth Privacy feature enabled (:kconfig:option:`CONFIG_BT_PRIVACY`).
+  If the Android 14 device gets disconnected from the HID device without explicit user action triggered in the Bluetooth UI from Android Settings (e.g. due to the Connection Timeout as a result of the HID device reboot), and this disconnect event occurs in the same connection session in which successful Bluetooth pairing and bonding have happened, the smartphone cannot reconnect with this target HID device automatically in the background and on the explicit user intent via the Bluetooth UI.
+  See the following list of affected samples and applications:
+
+  * :ref:`peripheral_hids_mouse`
+  * :ref:`fast_pair_input_device`
+  * Fast Pair configurations of the :ref:`nrf_desktop` application
+
+  **Workaround** After successful Bluetooth pairing and bonding, trigger manual disconnection on the Android device using the Bluetooth UI from Android Settings and then reconnect to the HID device.
```

Comment:
```suggestion
  **Workaround** After successful Bluetooth pairing and bonding, manually disconnect the Android device using the Bluetooth UI from Android Settings, and then reconnect to the HID device.
```

Reply:
Settings -> settings to be consistent with your previous suggestions